.yarn/patches/@metamask-assets-controllers-npm-46.0.1-3c56f3157c.patch

```diff
@@ -19,3 +19,50 @@ index 017fb94055b64f99c75f8d54b763a501bdd03e97..34396ba143e3ebcb04fa2c80f7a35d1a
      // We want to ensure that the CID is v1 (https://docs.ipfs.io/concepts/content-addressing/#identifier-formats)
      // because most cid v0s appear to be incompatible with IPFS subdomains
      return {
+diff --git a/dist/TokensController.cjs b/dist/TokensController.cjs
```

Comment:
Should we also update the .mjs file as well for webpack builds?

Reply:
usually updating the cjs file is enough when we create patch

Reply:
Hmm as long as this passes CI (webpack builds use mjs).

I'm guessing we will release a new version soon.